### ISSUE TITLE
Add .NET Client

### ISSUE THREAD

**Issue by hawkerm** [Mikeware/SpaceBattleArena#162]
@lee-11 and @mlhpdx I was looking at the network for Space Battle and stumbled upon [your fork](https://github.com/mlhpdx/SpaceBattleArena) adding .NET Core support!

Would you be interested in helping to submit a PR to add it to our official repo?

The main comments I have on the implementation are:
- [ ] Use PascalCasing for methods to align to C# naming conventions
- [ ] Also allow typing 'quit' in the console to exit the program (I believe this should make it work to use with F5 in VS Code as well, as Ctrl+C doesn't work there).
- [ ] Copy the java example ships for the test ship directory as individual ship files.

**Comment by mlhpdx** [comment]
Yep. I was going to get back to this again soon.  Still teaching Unit 3 right now, but I'll find some time to make this port a little nicer and if it makes sense PR it.
